File: src/main/java/it/contrader/dto/ParkingplaceDTO.java
package it.contrader.dto;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;

import it.contrader.model.Floor;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIdentityInfo(generator=ObjectIdGenerators.PropertyGenerator.class, property="id")
public class ParkingplaceDTO {
	
	private long id;
	
	private int numberplace;
	//@JsonProperty("floor")
	private Floor floor;
}

File: src/dto/parkingplacedto.ts
import { FloorDTO } from 'src/dto/floordto';



export class ParkingplaceDTO{


    id: number;

    numberplace: number;

    floor: FloorDTO = new FloorDTO();

 /*    constructor(){
        this.id = null;
        this.numberplace= null;
        this.floor= new FloorDTO();
    } */
}